now/cloud_manager.py

```diff
@@ -18,37 +18,43 @@ def create_local_cluster(kind_path):
     if err and 'No kind clusters' not in err.decode('utf-8'):
         print(err.decode('utf-8'))
         exit()
-    if 'jina-now' in out.decode('utf-8'):
+    cluster_name = 'jina-now'
+    if cluster_name in out.decode('utf-8'):
         questions = [
             {
                 'type': 'list',
                 'name': 'proceed',
                 'message': 'The local cluster is running already. '
                 'Should it be recreated?',
                 'choices': [
-                    {'name': '⛔ no - stop', 'value': False},
-                    {'name': '✅ yes - recreate', 'value': True},
+                    {'name': '⛔ no', 'value': False},
+                    {'name': '✅ yes', 'value': True},
                 ],
             },
         ]
         recreate = prompt_plus(questions, 'proceed')
         if recreate:
             with yaspin(text="Remove local cluster", color="green") as spinner:
-                cmd(f'{kind_path} delete clusters jina-now')
+                cmd(f'{kind_path} delete clusters {cluster_name}')
                 spinner.ok('💀')
         else:
             cowsay.cow('see you soon 👋')
             exit(0)
     with yaspin(text="Setting up local cluster", color="green") as spinner:
         cmd(
-            f'{kind_path} create cluster --name jina-now --config {cur_dir}/kind.yml',
+            f'{kind_path} create cluster --name {cluster_name} --config {cur_dir}/kind.yml',
         )
         spinner.ok("📦")
 
 
 def is_local_cluster(**kwargs):
+    command = f'{kwargs["kubectl_path"]} get nodes -o json'
     out, error = cmd(f'{kwargs["kubectl_path"]} get nodes -o json')
-    out = json.loads(out)
+    try:
+        out = json.loads(out)
+    except:
+        print(f'Command {command} gives the following error: {out} {error}')
+        exit(1)
```

Comment:
Why do we need to put it in try block?

Reply:
in case, the command does not work, an error is thrown and the output is not a json.

Reply:
Then we can just print the error component in string format: 
```suggestion
    try:
        out = json.loads(out)
    except:
        print(f'Command {command} gives the following error: {error.decode("utf-8")}')
        exit(1)
```